ui/src/app/layout/createtopic/createtopic.component.html

```diff
@@ -48,10 +48,10 @@ <h2 class="text-muted">{{ 'Create Topic' | translate }}</h2>
                             <label style="margin-right: 1em">{{ 'Topic Type' | translate }}</label>
                             <select class="form-control" name="topicType" [(ngModel)]="topicType"
                                     (change)="topicNameSpinner.while(suggestTopicName())">
-                                <option value="EVENTS">Events</option>
-                                <option value="DATA">Master Data</option>
-                                <option value="COMMANDS">Commands</option>
-                                <option value="INTERNAL">App internal</option>
+                                <option value="EVENTS">{{ 'Events' | translate }}</option>
+                                <option value="DATA">{{ 'Master Data' | translate }}</option>
+                                <option value="COMMANDS">{{ 'Commands' | translate }}</option>
+                                <option value="INTERNAL">{{ 'App internal' | translate }}</option>
```

Comment:
I'd prefer using the enum values as translation keys, this will make things easier... (`'EVENTS' | translate`)